pkg/convert/pot.go

```diff
@@ -0,0 +1,132 @@
+package convert
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+type PoTag string
+
+const (
+	MsgId  PoTag = "msgid"
+	MsgStr PoTag = "msgstr"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	if _, err := fmt.Fprintf(&b, "\"Language: %s\n", m.Language); err != nil {
+		return nil, fmt.Errorf("write language: %w", err)
+	}
+
+	for _, message := range m.Messages {
+		if message.Description != "" {
+			if _, err := fmt.Fprintf(&b, "#. %s\n", message.Description); err != nil {
+				return nil, fmt.Errorf("write description: %w", err)
+			}
+		}
+
+		if message.Fuzzy {
+			if _, err := fmt.Fprint(&b, "#, fuzzy\n"); err != nil {
+				return nil, fmt.Errorf("write fuzzy: %w", err)
+			}
+		}
+
+		msgId, err := formatPoTags(message.ID, MsgId)
+		if err != nil {
+			return nil, fmt.Errorf("format msgid: %w", err)
+		}
+
+		if _, err = fmt.Fprint(&b, msgId); err != nil {
+			return nil, fmt.Errorf("write msgid: %w", err)
+		}
+
+		msgStr, err := formatPoTags(message.Message, MsgStr)
+		if err != nil {
+			return nil, fmt.Errorf("format msgstr: %w", err)
+		}
+
+		if _, err = fmt.Fprint(&b, msgStr); err != nil {
+			return nil, fmt.Errorf("write msgstr: %w", err)
+		}
+
+		if _, err = fmt.Fprint(&b, "\n"); err != nil {
```

Comment:
This always creates an empty line at the end of the file. It's better to add an empty line between messages.

```suggestion
	for i, message := range m.Messages {
		if i > 0 {
			if _, err = fmt.Fprint(&b, "\n"); err != nil {
```